backend/api/models/document.py

```diff
@@ -4,24 +4,18 @@
 from .factory import Factory
 
 from users.models import CustomUser
+from .const import DocumentDisplayStatusConst
 
 
 class DocumentDisplayStatusEnum:
 
-    CHOICES = list(
-        enumerate(
-            [
-                "已檢舉",
-                "已排程稽查",
-                "陳述意見期",
-                "已勒令停工",
-                "已發函斷電",
-                "已排程拆除",
-                "已拆除",
-                "等待新事證",
-            ]
-        )
-    )
+    CHOICES = list(enumerate(DocumentDisplayStatusConst.STATUS_LIST))
+    INDICES = {val: idx for idx, val in CHOICES}
+
+
+class DocumentDisplayStatusEnrichmentEnum:
```

Comment:
Hmmm interesting. @EagleC0318  , are we really use this class? It seems to not be used by anyone.

Reply:
Yes you're right. 
It's something I added during the refactoring, but eventually it's not used anymore.
We can just remove it.